pysph/sph/wc/crksph.py

```diff
@@ -625,11 +630,10 @@ def loop(self, d_idx, s_idx, d_m, s_m, d_rho, s_rho, d_p, s_p, d_cs, s_cs,
         Qi = rhoi * (-Cl*ci*mui + Cq*mui*mui)
         Qj = rhoj * (-Cl*cj*muj + Cq*muj*muj)
 
-        fac = -(0.5/mi) * Vi * Vj * (pi + pj + Qi + Qj)
-
-        d_au[d_idx] += fac * DWIJ[0]
-        d_av[d_idx] += fac * DWIJ[1]
-        d_aw[d_idx] += fac * DWIJ[2]
+        fac = -(1.0/mi) * Vi * Vj * (pi + pj + Qi + Qj)
+        d_au[d_idx] += fac * (DWIJ[0])  # - DWJ[0])
```

Comment:
Why is the anti-symmetric component commented out?  Is that a mistake?

Reply:
DWIJ = 0.5*(DWI - DWJ)

i was just testing earlier, will remove the comment